### ISSUE TITLE
Riak example recommends running sshd in container

### ISSUE THREAD

**Issue by bfirsh** [docker/docker#6946]
https://docs.docker.com/examples/running_riak_service/

http://jpetazzo.github.io/2014/06/23/docker-ssh-considered-evil/

**Comment by thieman** [comment]
This should have been closed once https://github.com/docker/docker/pull/11870 was merged.

**Comment by thaJeztah** [comment]
Good catch! I just spotted that there's still one reference to "SSH" in the description of the document, that might need to be fixed at some point (if you feel like creating a PR, that would be great ;-))

@moxiegirl any idea why that change wasn't merged to docs yet? Or just haven't come round to it?

I'm going to close this issue, because it has been resolved in https://github.com/docker/docker/pull/11870

Thanks for spotting this!
